src/app/georeferencer/qgsgeorefdatapoint.h

```diff
@@ -28,7 +29,7 @@ class QgsGeorefDataPoint : public QObject
     //! constructor
     QgsGeorefDataPoint( QgsMapCanvas *srcCanvas, QgsMapCanvas *dstCanvas,
                         const QgsPointXY &pixelCoords, const QgsPointXY &mapCoords,
-                        bool enable );
+                        const QString &wktProj, bool enable );
```

Comment:
Same as above, avoid passing on a string, you can pass on a QgsCoordinateReferenceSystem object instead.

Reply:
I wasn't sure if it was better to store a string or the crs object, as this meant more imports but no conversion needed.

Reply:
done